framework/src/main/java/org/checkerframework/common/subtyping/SubtypingAnnotatedTypeFactory.java

```diff
@@ -49,17 +49,30 @@ protected AnnotationClassLoader createAnnotationClassLoader() {
         // load individually named qualifiers
         if (qualNames != null) {
             for (String qualName : qualNames.split(",")) {
-                qualSet.add(loader.loadExternalAnnotationClass(qualName));
+                Class<? extends Annotation> anno = loader.loadExternalAnnotationClass(qualName);
+                if (anno == null) {
+                    throw new UserError("qualifier specified in -Aquals not found: " + qualName);
+                }
+                qualSet.add(anno);
             }
         }
 
         // load directories of qualifiers
         if (qualDirectories != null) {
             for (String dirName : qualDirectories.split(":")) {
-                qualSet.addAll(loader.loadExternalAnnotationClassesFromDirectory(dirName));
+                Set<Class<? extends Annotation>> annos =
+                        loader.loadExternalAnnotationClassesFromDirectory(dirName);
+                if (annos.isEmpty()) {
+                    throw new UserError("No qualifiers found in " + dirName);
+                }
+                qualSet.addAll(annos);
             }
         }
 
+        if (qualSet.isEmpty()) {
+            throw new UserError("SubtypingChecker: no qualifiers specified");
```

Comment:
The three user errors have different shapes, one starting with a lower case letter, the next with upper case, and this final one includes the checker name.
Always including the checker name seems useful.

Reply:
I added more specific command-line argument references, and I think that improves the messages.
I also fixed the capitalization.
Thanks for those suggestions.

The first two messages refer to a command-line argument the user provided incorrectly, so I think that the checker name is not necessary there, and I want to keep error messages short when possible.
The last is about command-line arguments that are missing, which is why I provided the checker name there.
Do you still think it's important to always give the checker name?